specification/overview.md

```diff
@@ -193,20 +193,23 @@ OpenTelemetry defines the naming convention for metric names as well as a
 well-known metric names in [Semantic Conventions](data-semantic-conventions.md)
 document.
 
-## DistributedContext
+## CorrelationContext
 
-The **DistributedContext** exists to store labels that describe the context of an operation an application performs. It is intended to enable context that are custom to the application or integrations in contrast to other contexts, such as `SpanContext`. Only one **DistributedContext** should be associated with any particular operation.
+In addition to trace propagation, OpenTelemetry provides a simple mechanism for propagating indexes, called **CorrelationContext**. The **CorrelationContext** is intended for indexing observability events in one service with attributes provided by a prior service in the same transaction. This helps to establish a causal relationship between these events. For example, determining that a particular browser version is associated with a failure in an image processing service.
```

Comment:
Replace "indexes" by "set of name/value entries"?

Reply:
done